common/meta/app/src/schema/table.rs

```diff
@@ -174,8 +174,8 @@ pub struct TableMeta {
     pub engine: String,
     pub engine_options: BTreeMap<String, String>,
     pub options: BTreeMap<String, String>,
-    pub cluster_key: Option<String>,
-    // The vector of cluster keys.
+    pub current_cluster_key: Option<String>,
```

Comment:
I think default_cluster_key is better than current_cluster_key. Because we have a default_cluster_key_id